desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -103,18 +110,19 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
       else
         resolve()
 
+  # Lock the form- action.finally() callback executes when form submission is complete regardless of success
   loadingLock: ($element, action) ->
     return if $element.hasClass('is-loading')
     $element.addClass 'is-loading'
-    action().finally => $element.removeClass 'is-loading'
+    action().finally => $element.removeClass 'is-loading' unless @comboForm
```

Comment:
So this is the only line that confuses me- I don't know why it works in the following situation:
- at the register+bid form
- set cc date back to 2017 so the registration will fail
- submit

Shouldn't `finally()` be NOT removing the loading spinner class from the button? It does, which is the desired behavior, but It seems like it shouldn't.

(the reason to not remove it is that this will affect the following submission as well)